### DISCUSSION TITLE
Trouble staying logged in

### DISCUSSION THREAD

**Discussion by andlil** [Budibase/budibase#14036] [Help] [answered]
I have problems staying logged in to the app, after about an hour or two I have to login again. I thought that #12872 which was discussed in #13263 would prevent me from getting logged out. 

Is not persistent authentication done through cookies (or am I old fashioned?)?

My app is selfhosted at home (Docker) behind Traefik, I limit access to the app by IP-address so access from work where I and a colleague is from one IP-address and from mobile I use a VPN to access my home network.

Any ideas?

//A

**Comment by mike12345567** [comment, marked as answer]
Hi @andlil - are you using multiple devices to log into the same user account? Budibase only allows app builder users to login on a single device, if you attempt to use the same credentials on another device then all other instances of that account will be logged out.

This is to avoid issues with tracking user interactions, if the same user is interacting with and building multiple apps/the same app across multiple devices Budibase can run into issues tracking these updates.

**Reply by andlil** [reply]
Yes, different devices, same login. Now that you say it I should have considered that possibility. 

I will make another login for use on my mobile device then.

Thank you!
//A
